src/api/fastq.ts

```diff
@@ -0,0 +1,34 @@
+import { API } from '@aws-amplify/api';
```

Comment:
Having said; could please keep this. We might need it in future...

Reply:
Yuppp 🫡